score/ipython/score_calc_0.1.ipynb

```diff
@@ -70,7 +72,7 @@
    "outputs": [],
    "source": [
     "# strip calculations\n",
-    "df = df[[\"ID\", \"score_a_percentile\", \"score_b_percentile\",\"score_a_top_percentile_25\",\"score_b_top_percentile_25\"]]"
+    "df = df[[\"ID\", \"ACSTOTPOP\", \"score_a\",\"score_b\", \"score_a_percentile\", \"score_b_percentile\",\"score_a_top_percentile_25\",\"score_b_top_percentile_25\"]]"
```

Comment:
what does this value mean: `ACSTOTPOP`?

Reply:
census block group total population - we should do a quick style guide around naming practices for variables to all get on the same page. (E.g., I've often worked on projects where we renamed all input data variables into our more readable style patterns.)

Creating this now! https://app.zenhub.com/workspaces/justice40-60993f6e05473d0010ec44e3/issues/usds/justice40-tool/189